### PR TITLE
My chapter

### PR DESCRIPTION
**Pull request by mongoose1729**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/chuck-norris/chucknorris.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+You pull out your cell phone and text Chuck Norris to ask what he would do.
```
Comment (casey-collab) [inline english/chuck-norris/chucknorris.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by reyesh** [comment]
@casey-collab

File: english/chuck-norris/chucknorris.md (hunk 1 above)
Comment (casey-collab) [inline english/chuck-norris/chucknorris.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@reyesh, I don't see any new changes. If you made some changes locally, remember that you'll need to push them before they show up in the pull request.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**